### PR TITLE
feat: support Mac OS build for docker-compose/goloop2moonbeam

### PR DESCRIPTION
**Pull request by trmaphi**
- In our development, goloop node is confirmed unstable during long run and we used gochain for long run.
- Contracts deployments and transfer still work.
- We tried to run only goloop container with 100 transfer ICX transactions concurrently and it’s ok.
- We changes icon.genesis.json with new attributes, goloop container still die out.
```
        "roundLimitFactor": "0x6",
        "commitTimeout": "0x3e8",
```

Problem with goloop containers on Mac Os, this problem doesn't occur on Ubuntu
- Using `docker-compose down -v --remove-orphans` in `make clean` cause
```
Traceback (most recent call last):
  File "requests/adapters.py", line 439, in send
  File "urllib3/connectionpool.py", line 726, in urlopen
  File "urllib3/util/retry.py", line 410, in increment
  File "urllib3/packages/six.py", line 735, in reraise
  File "urllib3/connectionpool.py", line 670, in urlopen
  File "urllib3/connectionpool.py", line 428, in _make_request
  File "urllib3/connectionpool.py", line 335, in _raise_timeout
urllib3.exceptions.ReadTimeoutError: UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=60)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "threading.py", line 950, in _bootstrap_inner
  File "threading.py", line 888, in run
  File "compose/cli/log_printer.py", line 202, in watch_events
  File "compose/project.py", line 626, in yield_loop
  File "compose/project.py", line 594, in build_container_event
  File "compose/container.py", line 44, in from_id
  File "docker/utils/decorators.py", line 19, in wrapped
  File "docker/api/container.py", line 774, in inspect_container
  File "docker/utils/decorators.py", line 46, in inner
  File "docker/api/client.py", line 237, in _get
  File "requests/sessions.py", line 543, in get
  File "requests/sessions.py", line 530, in request
  File "requests/sessions.py", line 643, in send
  File "requests/adapters.py", line 529, in send
requests.exceptions.ReadTimeout: UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=60)

ERROR: for g2m_goloop  UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=70)
ERROR: An HTTP request took too long to complete. Retry with --verbose to obtain debug information.
If you encounter this issue regularly because of slow network conditions, consider setting COMPOSE_HTTP_TIMEOUT to a higher value (current value: 60).
make: *** [Makefile:9: run] Error 1
```
- in order to stop goloop container, I have to restart docker-engine and `make clean` again
- goloop container still receive transactions all transactions before goloop hange up receiver PENDING error. In this time, goloop node doesn't produce blocks, logs.

Proposed solution:
- replace goloop image with gochain

Added:
- gochian image doesn't work too

### CODE REVIEW

File: docker-compose/goloop2moonbeam/README.MD [hunk 1]
```diff
@@ -14,22 +16,35 @@ This document will walk you through setting up a Docker environment to demonstra
...
 ```
 
+For Mac OS, we recommend to use homebrew for prerequisites
+```
+brew install git make go dokcer docker-compose
+```
+
+## Computer specs
+
+At least 8 CPUs, 16 RAM.
```
Comment (canhlinh) [inline docker-compose/goloop2moonbeam/README.MD:34]: @trmaphi I don't think it costs so much ram ?

Comment (trmaphi) [reply docker-compose/goloop2moonbeam/README.MD]: For safety threshold

Comment (canhlinh) [reply docker-compose/goloop2moonbeam/README.MD]: 4g is enough I think so. Since, docker get all of ram the host will be slow

Comment (canhlinh) [reply docker-compose/goloop2moonbeam/README.MD]: Screen Shot 2021-09-03 at 15 22 33

**Comment by trmaphi** [comment]
Close as not needed anymore

**Pull request closed without merging**